tests/ui/int_plus_one.stderr

```diff
@@ -1,44 +1,44 @@
 error: Unnecessary `>= y + 1` or `x - 1 >=`
-  --> $DIR/int_plus_one.rs:7:5
+  --> $DIR/int_plus_one.rs:9:13
    |
-LL |     x >= y + 1;
-   |     ^^^^^^^^^^
+LL |     let _ = x >= y + 1;
+   |             ^^^^^^^^^^
    |
    = note: `-D clippy::int-plus-one` implied by `-D warnings`
 help: change `>= y + 1` to `> y` as shown
```

Comment:
Another case of "Not your fault, but while you're at it":

Can you change the help message of the lint to "change it to", to match the rules of a [`span_suggestion`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_errors/struct.Diagnostic.html#method.span_suggestion)

Reply:
Oh I never saw those rules before, thanks :D